Rubberduck.Core/UI/Settings/GeneralSettingsViewModel.cs

```diff
@@ -210,10 +213,11 @@ private void TransferSettingsToView(IGeneralSettings general, IHotkeySettings ho
             AutoSaveEnabled = general.IsAutoSaveEnabled;
             AutoSavePeriod = general.AutoSavePeriod;
             SelectedLogLevel = LogLevels.First(l => l.Ordinal == general.MinimumLogLevel);
-            
-            ExperimentalFeatures = RubberduckIoCInstaller.AssembliesToRegister()
-                .SelectMany(s => s.DefinedTypes)
-                .Where(w => Attribute.IsDefined(w, typeof(ExperimentalAttribute)))
+
+            //RubberduckIoCInstaller.AssembliesToRegister()
+            //    .SelectMany(s => s.DefinedTypes)
+            //    .Where(w => Attribute.IsDefined(w, typeof(ExperimentalAttribute)))
+            ExperimentalFeatures = _experimentalFeatureTypes
```

Comment:
Delete the commented code. 👍

Reply:
still needs to be migrated to the IoCInstaller IIRC